File: features/user/topchat/src/main/java/com/tokopedia/topchat/chatlist/domain/pojo/reply/Message.java
package com.tokopedia.topchat.chatlist.domain.pojo.reply;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by stevenfredian on 11/1/17.
 */

public class Message {

    @SerializedName("censored_reply")
    @Expose
    private String censoredReply;
    @SerializedName("original_reply")
    @Expose
    private String originalReply;
    @SerializedName("timestamp")
    @Expose
    private String timestamp;
    @SerializedName("timestamp_fmt")
    @Expose
    private String timestampFmt;
    @SerializedName("timestamp_unix_nano")
    @Expose
    private String timeStampUnixNano;
    @SerializedName("timestamp_unix")
    @Expose
    private String timeStampUnix;

    public String getCensoredReply() {
        return censoredReply;
    }

    public String getOriginalReply() {
        return originalReply;
    }

    public String getTimestamp() {
        return timestamp;
    }

    public void setTimestamp(String timestamp) {
        this.timestamp = timestamp;
    }

    public String getTimestampFmt() {
        return timestampFmt;
    }

    public String getTimeStampUnix() {
        return timeStampUnix;
    }

    public String getTimeStampUnixNano() {
        return timeStampUnixNano;
    }
}

File: features/communication/topchat/src/main/java/com/tokopedia/topchat/chatlist/domain/pojo/reply/Message.kt
package com.tokopedia.topchat.chatlist.domain.pojo.reply

import com.google.gson.annotations.SerializedName

class Message {
    @SerializedName("censored_reply")
    val censoredReply: String = ""

    @SerializedName("original_reply")
    val originalReply: String = ""

    @SerializedName("timestamp")
    var timestamp: String = ""

    @SerializedName("timestamp_fmt")
    val timestampFmt: String = ""

    @SerializedName("timestamp_unix_nano")
    val timeStampUnixNano: String = ""

    @SerializedName("timestamp_unix")
    val timeStampUnix: String = ""
}